lib/susedistribution.pm

```diff
@@ -694,6 +739,8 @@ sub activate_console {
     elsif ($name =~ /log|tunnel/) {
         $user = 'root';
     }
+    # Use ssh for generalhw(ssh/noVNC) for given consoles
```

Comment:
can you also explain *why*? Can we try to avoid this at all somehow? Maybe just die with an error if this is called here for something which has *only* ssh based consoles and the type differs?

Reply:
Not sure how I could give more details. Without VNC, we use ssh consoles.
IIUC, you would like to set console `type` elsewhere?

Reply:
No, "root-console" is the right one already. I think we use the root console over VNC in the case of other remote backends but handling it in activate_console does somehow not look right to me. Sorry if I can not be more clear here. If there is nothing else we can leave it as you have it currently.